src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3235,7 +3235,6 @@ private bool IsFieldDeclaration(bool isEvent, bool isGlobalScriptLevel)
             {
                 case SyntaxKind.DotToken:                   // Goo.     explicit
                 case SyntaxKind.ColonColonToken:            // Goo::    explicit
-                case SyntaxKind.DotDotToken:                // Goo..    explicit
```

Comment:
Most parser code got simpler.  Instead of dealing with DotDot as if it was an errant Dot that they need to split, tehy just handle 'dot' like normal.  only the code that explicitly cares about a `..` token needs to handle and synthesize it.